Documentation-rendertest/Typesetting/Index.rst

```diff
@@ -1,170 +1,132 @@
 .. include:: /Includes.rst.txt
 
-==============
+..  _typesetting:
+
+===========
 Typesetting
-==============
+===========
 
-.. contents:: This page
-   :backlinks: top
-   :class: compact-list
-   :local:
+..  contents:: Table of contents
 
+Introduction
+============
 
+This is a typesetting test in reStructuredText. You will be required to demonstrate various formatting options including:
 
-==============
-Typesetting H1
-==============
+- **Headers**
+- *Emphasis*
+- `Inline code`
+- Lists
+- Blockquotes
+- Tables
 
-Typesetting H2
+..  _headers:
+
+Headers
+=======
+
+There are multiple levels of headers available:
+
+Level 2 Header
 ==============
 
-Typesetting H3
+Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod
+tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.
+
+Level 3 Header
 --------------
 
-Typesetting H4
+At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd
+gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
+Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod
+tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.
+
+Level 4 Header
 ~~~~~~~~~~~~~~
 
-Typesetting H5
-++++++++++++++
+At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd
+gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
```

Comment:
Please also define a fifth level. I remember, somewhere in the docs I saw that deep level.